torch/_inductor/autotune_process.py

```diff
@@ -218,15 +219,15 @@ class TuningProcessPool:
     processes: Optional[queue.Queue[TuningProcess]] = None
     executor: Optional[ThreadPoolExecutor] = None
 
-    def initialize(self) -> None:
+    def initialize(self, device_type: str = "cuda") -> None:
```

Comment:
In terms of the device type, you can leverage `V.graph` but don't need to add a dedicated parameter.